certs/api/endpoint.go

```diff
@@ -20,12 +20,14 @@ func issueCert(svc certs.Service) endpoint.Endpoint {
 		if err != nil {
```

Comment:
I like this proposition.

Reply:
Is this resolved?

Reply:
I'm not sure but I think that this remark was previous to the renaming of `days_valid` into `hours_valid`. IMO it's better to use `hours_valid`, it's self documenting. `ttl` is a Vault therminology.

Reply:
I agree that `hours_valid`, while a little clumsy, is more self-documenting. However, since we are using it as a string, we can include a time determinant such as "h", "m" or "s", so TTL is probably a better naming. Also, since we don't control the aforementioned time determinant, `hours` in the name is actually misleading.

Reply:
Right. What I mean is that we are using `TTL`  in the pki layer but we don't necessaraly have to follow this therminology for the HTTP API. However we are using `KeyBytes` and `KeyType` so maybe I can change it.

Reply:
ok, let me rename then